### PR TITLE
Issue 5 and 6: Add image filtering by tag and ability to delete images

### PR DESCRIPTION
**Pull request by jeremykirc**

### CODE REVIEW

**Comment by jeremykirc** [comment]
[Issue 5: Images by Tag](https://trello.com/c/MN0zJKRz/999-issue-5-images-by-tag)

**Comment by jeremykirc** [comment]
[Issue 6: Delete Image](https://trello.com/c/OuzMCLZG/994-issue-6-delete-image)

**Comment by aman-singh** [comment]
Hey, it's nice to make 1 PR per change, keeps PRs small and focussed on a change.
Also if there are changes in a PR that are not part of the same unit of work then its good to split those out in different commits with separate descriptions.
I understand its quicker this way but you're doing great on time!

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -12,24 +12,24 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
   # Test create.
```
Comment (aman-singh) [inline test/controllers/images_controller_test.rb:13]: tests for destroy?

**Comment by aman-singh** [comment]
Changes look good! Please add those tests for destroy image and this is good to go!

**Review by aman-singh — APPROVED** [review]

**Pull request merged**